upi/openstack/network.yaml

```diff
@@ -16,10 +16,18 @@
       name: "{{ os_network }}"
     when: os_networking_type != "CiscoACI"
 
+  - name: 'list network'
+    command:
+      cmd: "openstack network list --name {{ os_network }} --tag {{ cluster_id_tag }}"
+    when: os_networking_type == "CiscoACI"
+    register: cluster_network
+
```

Comment:
The check here is only for the name and tag, the requirement is also to check if the other attributes used during network-create are the same as the existing network. If a network with the name exists, but the attributes do not match I think we should error out.

Reply:
Addressed. Neutron CLI  is used to get details of apic:epg contract master attributes of network.